.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: test
+
+on:
+  push:
+    tags:
+      - test*
+
+permissions:
+  contents: write
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: referer-json
+      - name: Get datetime
+        id: datetime
+        run: echo "datetime=$(date "+%Y-%m-%d-%H%M%S")" >> "$GITHUB_ENV"  
+      - name: Convert yaml to json
+        run: python .github/workflows/lib/yaml_to_json.py resources/referers.yml resources/referers-$datetime.json
+      - name: Prepare latest json
+        run: cp resources/referers-$datetime.json resources/referers-latest.json
+      - name: Commit and push new json files
+        run: |
+          git config --local user.email "github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+          git add resources/referers-$datetime.json resources/referers-latest.json
+          git commit -m "Add referers-$datetime.json"
+          git push
```

Comment:
Does this push to a branch?  And if yes, which branch?

Can you give an example of the URL that we then use for testing in the pipeline?

Reply:
this pushes to `referer-json` branch, pointed at https://github.com/snowplow-referer-parser/referer-parser/pull/240/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R17

an example url that I used in my test is
```
https://raw.githubusercontent.com/snowplow-referer-parser/referer-parser/refs/heads/referer-json/resources/referers-2025-03-14-113945.json
```

Reply:
OK understood.  I like that.

Are you also going to delete the `pull_request.yml` github action, which I think you only added as a test/demo?

Reply:
And sorry one more thing... please could you add a code comment to this github action, to remind us what url to look for.

Reply:
> Are you also going to delete the pull_request.yml github action, which I think you only added as a test/demo?

I wasn't planning to delete it as it makes sure any change to `referers.yml` in a PR is valid and the file can still be converted to json.

I'll add a comment to the github action.

Reply:
added another step to print the url e.g. https://github.com/snowplow-referer-parser/referer-parser/actions/runs/13857313589/job/38776935473#step:7:8 instead of a comment